store/tikv/latch/scheduler.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package latch
+
+const lockChanSize = 100
+
+// LatchesScheduler is used to schedule latches for transactions.
+type LatchesScheduler struct {
+	latches *Latches
+	lockCh  chan *Lock
+}
+
+//NewLatchesScheduler create the LatchesScheduler.
+func NewLatchesScheduler(size int) *LatchesScheduler {
```

Comment:
s/NewLatchesScheduler/NewScheduler
Outside side the package, we've already get the package name `latch.NewScheduler()`